src/styles/index.ts

```diff
@@ -5848,6 +5848,27 @@ const styles = (theme: ThemeColors) =>
         thumbnailImageContainerHighlight: {
             backgroundColor: theme.highlightBG,
         },
+
+        TestToolsNavigatorOuterView: (shouldUseNarrowLayout: boolean) => ({
+            flex: 1,
+            justifyContent: shouldUseNarrowLayout ? 'flex-end' : 'center',
+            alignItems: 'center',
+        }),
+
+        TestToolsNavigatorInnerView: (shouldUseNarrowLayout: boolean, isAuthenticated: boolean) => {
```

Comment:
These two style name are different with our current convention (camel case), let's change their name to match the current convention then?
I think it can be: `getTestToolNavigatorOuterView` or so.